### ISSUE TITLE
Image sharpness issues such as lines and fonts on fabric js 1.7X version

### ISSUE THREAD

**Issue by Kennymoon** [kangax/fabric.js#4387]
Dear, 

Our team created an application using the 1.5.x version of fabric js.
In 1.5.x version, when one object attribute is changed, only the corresponding object should be re-rendered, but all the objects contained in the Canvas are re-rendered, resulting in unnecessary memory use and operation, which causes performance degradation.
Therefore, in order to improve the screen rendering performance, we decided to upgrade application program made in 1.5.x version to 1.7.x version in order to utilize object cache function provided in fabric js 1.7.x version. 

I implemented a simple label control and expressed it on the Canvas. I found that the text displayed in the 1.5.x version of Canvas is vivid, but the clarity of the text on the Canvas upgraded to 1.7.x is low.

This phenomenon is noticeable when the font size is 12 px or less. (The test environment is IE version 11)

In summary, 1.5.x is a relatively clear representation of fonts, and 1.7.x is fuzzy. It is relatively clear on the edge of the screen, but it becomes blurred toward the center area.
(Chrome browser has low clarity in all areas) 

Anybody can help me to solve the problem ?

**Comment by asturur** [comment]
As per issue #4113 this is an issue.
Please keep the topic there

**Comment by Kennymoon** [comment]
thanks
